spec/message.abnf

```diff
@@ -91,9 +91,9 @@ name-start = ALPHA / "_"
            / %xC0-D6 / %xD8-F6 / %xF8-2FF
            / %x370-37D / %x37F-1FFF / %x200C-200D
            / %x2070-218F / %x2C00-2FEF / %x3001-D7FF
-           / %xF900-FDCF / %xFDF0-FFFD / %x10000-EFFFF
+           / %xF900-FDCF / %xFDF0-FFFC / %x10000-EFFFF
 name-char  = name-start / DIGIT / "-" / "."
-           / %xB7 / %x300-36F / %x203F-2040
+           / %xB7 / %x300-36F / %x203F-2040 / %xFFFD
```

Comment:
Lose this change. We don't want replacement characters anywwhere.

Reply:
```suggestion
           / %xB7 / %x300-36F / %x203F-2040
```